Firestore/Example/Tests/Integration/API/FIRWriteBatchTests.m

```diff
@@ -35,6 +35,23 @@ - (void)testSupportEmptyBatches {
   [self awaitExpectations];
 }
 
+- (void)testSupportEmptyCompletion {
```

Comment:
This name is slightly off--it's not an "empty" completion, it's "nil". Also this doesn't really name what supports no completion. A better name would be something like `testCommitWithoutCompletionHandler`.

Reply:
done